pkg/activator/throttler.go

```diff
@@ -137,7 +137,7 @@ func (t *Throttler) UpdateCapacity(rev RevisionID, size int) error {
 	if err != nil {
 		return err
 	}
-	return t.updateCapacity(breaker, int(revision.Spec.ContainerConcurrency), size, t.activatorCount())
+	return t.updateCapacity(breaker, int(*revision.Spec.ContainerConcurrency), size, t.activatorCount())
```

Comment:
I think rather than blindly dereferencing this everywhere we should add `GetContainerConcurrency()`, which returns `0` when the field is not present.